lib/livebook_web/live/session_live.ex

```diff
@@ -887,19 +931,6 @@ defmodule LivebookWeb.SessionLive do
     {:noreply, socket}
   end
 
-  def handle_event("format_code", %{"code" => code}, socket) do
-    formatted =
-      try do
-        code
-        |> Code.format_string!()
-        |> IO.iodata_to_binary()
-      rescue
-        _ -> code
-      end
-
-    {:reply, %{code: formatted}, socket}
-  end
-
```

Comment:
Was it intended to remove this whole event?

Reply:
Yup, I suggested that because it's a leftover :D